docs/index.rst

```diff
@@ -33,28 +33,35 @@ The vision is that it grows over time as the teams and governance structure evol
 
 .. toctree::
    :maxdepth: 2
-   :caption: Policies
+   :caption: Community leadership
    :hidden:
 
-   policies/code_of_conduct/code_of_conduct
- 
+   community_leadership/community_leadership
 
 .. toctree::
    :maxdepth: 2
    :caption: Policies
    :hidden:
 
+   policies/code_of_conduct/code_of_conduct
    policies/code_of_conduct/code_of_conduct_breaches
-
+   policies/code_of_conduct/tools_and_resources
+   policies/request_for_proposals_(RFP)_process
```

Comment:
Hey @RCheesley,

There's an error here. The [RFP section](https://github.com/mautic/mautic-community-handbook/pull/277/files#diff-62bf393018ca81243daf7caef308e14c5881b64113e560fb704ed018b46da605) is moved from policies to teams/product_team folder, but there's no change in toctree.

`make html` complained that it's not inlcluded in any toctree and building is failed: `/var/www/html/docs/teams/product_team/request_for_proposals_(RFP)_process.rst: WARNING: document isn't included in any toctree`

Based on my findings and as a background:

- In current state, RFP section belongs to `teams/product_team`.
- [Your comment](https://github.com/mautic/mautic-community-handbook/pull/247#pullrequestreview-2412204421) in the PR suggested to move it to policy.

Reply:
I've moved it to the policy section, as that's where it should live, will make a PR to fix the error. There's also a bunch of RST warnings in the build logs as well which we should address (mostly duplicated titles).